package.json

```diff
@@ -57,28 +57,21 @@
     "url": "https://github.com/sollenne/angular-fittext/issues"
   },
   "homepage": "https://github.com/sollenne/angular-fittext#readme",
-  "dependencies": {
-    "ajv": "^5.5.0",
-    "core-js": "^2.5.1",
-    "rxjs": "^5.5.2",
-    "zone.js": "^0.8.18"
-  },
   "peerDependencies": {
-    "@angular/common": ">=4.0.0 <6.0.0",
-    "@angular/compiler": ">=4.0.0 <6.0.0",
-    "@angular/core": ">=4.0.0 <6.0.0",
-    "@angular/forms": ">=4.0.0 <6.0.0",
-    "@angular/platform-browser": ">=4.0.0 <6.0.0",
-    "@angular/platform-browser-dynamic": ">=4.0.0 <6.0.0"
+    "@angular/core": "^6.0.0"
```

Comment:
This would preclude anything under 6.0.0, we still need to support 5. change this and i'll merge.